.github/workflows/ci.yml

```diff
@@ -55,6 +55,7 @@ jobs:
           fi
           npm install
           npm start &
+          until curl http://localhost:8545/ -X POST -H "Content-Type: application/json" --data '{"method":"eth_chainId","params":[],"id":1,"jsonrpc":"2.0"}'; do sleep 1 ; done
```

Comment:
🤞

Reply:
So, as I understand:
- the Linux version gets to log about starting the server https://github.com/status-im/nim-codex/actions/runs/4820668186/jobs/8587190962?pr=407#step:6:289
- the Macos version is seemingly stuck at downloading https://github.com/status-im/nim-codex/actions/runs/4820668186/jobs/8587191172?pr=407#step:6:206
- but than it magically passes the test and moves to the next phase https://github.com/status-im/nim-codex/actions/runs/4820668186/jobs/8587191172?pr=407#step:6:262